README.md

```diff
@@ -1,2 +1,53 @@
 # SMT Lean
-This project is inspired by [SMTCoq](https://smtcoq.github.io/) and aims to provide Lean tactics that discharge goals into SMT solvers. This project is in its early design/development phase and is not recommended for use.
+This project is inspired by [SMTCoq](https://smtcoq.github.io/) and aims to
+provide Lean tactics that discharge goals into SMT solvers. This project is
+in its early design/development phase and is not recommended for use.
+
+# Requirements
+`lean-smt` communicates with SMT solvers via the SMT-LIB textual interface.
+Therefore, it requires a supported SMT solver to be installed. Currently,
+`lean-smt` supports two SMT solvers:
+- [`cvc5`](https://cvc5.github.io) (default).
+- [`z3`](https://github.com/Z3Prover/z3).
+
+# Usage
+To use the `lean-smt` in your project. Add the following line to your list of
+dependencies in `lakefile.lean`:
+```lean
+require smt from git "https://github.com/ufmg-smite/lean-smt.git"@"main"
```

Comment:
Hi @tydeu! From previous discussion, I thought Lake was moving towards reproducibility, so I was surprised to see that this works now. Is it supposed to auto-update, or is the idea that the current `main` commit is persisted to a lockfile and will not change until we run `lake update` or something like that?

Reply:
Oh, [README](https://github.com/leanprover/lake#adding-dependencies) explains it.